jsk_denso_robot/sample_pack/canon_camera/src/camera_ctrl_server.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
```

Comment:
use python2 for melodic and python3 for noetic are preferred, see http://wiki.ros.org/UsingPython3/SourceCodeChanges#Changing_shebangs